### ISSUE TITLE
When is the SPF stream finished? Is Content-Length enough?

### ISSUE THREAD

**Issue by stain** [w3ctag/packaging-on-the-web#18]
As there is no end marker of Streamable Package Format (SPF) - how do we know the stream is finished and there are no more files, and that the connection was not broken for other reasons?

This could presumably be solved at the HTTP level - but could also be an issue for the file format. How do I now a .pack file is not half-way written to disk?

In the [SPF examples](http://w3ctag.github.io/packaging-on-the-web/#package-header) there are no `Content-Length` header. Presumably this can be difficult to know in advanced for a truly streaming server, in which case `Transfer-Encoding` SHOULD be used with a chunked transfer coding.

See https://tools.ietf.org/html/rfc7230#section-3.3.3

**Comment by ylafon** [comment]
Yes, there might be a need for an empty entry (like two delimiters in a row) to indicate the end of the package, similar to the last-chunk indicator in HTTP/1.1
